create-chroot.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e -u
+
+cd "$(dirname "$0")"
+
+if [ "${HOMEWORLD_CHROOT:-}" = "" -o ! -e "$(dirname HOMEWORLD_CHROOT)" ]
+then
+    echo "invalid path to chroot: ${HOMEWORLD_CHROOT:-}" 1>&2
+    echo '(have you populated $HOMEWORLD_CHROOT?)'
+    echo '(try export HOMEWORLD_CHROOT=$HOME/chroot)'
+    exit 1
+fi
+
+if [ -e "${HOMEWORLD_CHROOT}" ]
+then
+    echo "chroot already exists" 1>&2
+    exit 1
+fi
+
+mkdir "${HOMEWORLD_CHROOT}"
+sudo debootstrap --include="$(cat chroot-packages.list | grep -vE '^#' | tr '\n ' ',,' | sed 's/,$//')" stretch "${HOMEWORLD_CHROOT}" http://debian.csail.mit.edu/debian/
+sudo ln -sT "homeworld/building" "${HOMEWORLD_CHROOT}/h"
+sudo chroot "${HOMEWORLD_CHROOT}" useradd -m -u "$UID" "$USER" -s "/bin/bash"
```

Comment:
Why specify the uid of the new user?

Reply:
The idea was to make sure it matches the actual user, because the filesystem is shared. I feel like there might be another solution to this, but I can't remember what it is.